metadata.json

```diff
@@ -10,15 +10,15 @@
   "dependencies": [
     {
       "name": "WhatsARanjit/node_manager",
-      "version_requirement": ">= 0.7.3 < 0.8.0"
+      "version_requirement": ">= 0.7.3 < 1.0.0"
     },
     {
       "name": "puppetlabs/inifile",
-      "version_requirement": ">= 4.2.0 <= 4.3.0"
+      "version_requirement": ">= 1.0.0 < 5.0.0"
```

Comment:
Looks like Litmus' `install_module` tasks finds the _latest_ satisfying version of dependencies. So, I went ahead and manually installed `inifile` 1.0.0 and ran the acceptance tests and they seem to pass.

Reply:
```
root@prefab-lordship:~# cat /etc/puppetlabs/code/environments/production/modules/inifile/metadata.json
{
  "name": "puppetlabs-inifile",
  "version": "1.0.0",
  "source": "git://github.com/puppetlabs/puppetlabs-inifile.git",
...
```

Reply:
Also did a `git diff <rev1> <rev2> -- lib/puppet/type/ini_setting.rb` (since we only use `ini_setting` resource), and seems like between `1.0.0`, `2.0.0`, `3.0.0, `4.0.0` I couldn't find any breaking changes to the resource (in the context of how we use it).

Reply:
I'll go ahead and run the acceptance tests for each module version just to be safe.

Reply:
Cool, tests passed. This should be good to go.